yamdwe.py

```diff
@@ -49,8 +49,9 @@ def main():
     pages = importer.get_all_pages()
     print("Found %d pages to export..." % len(pages))
 
-    # Add a shameless "exported by yamdwe" note to the front page of the wiki
+    # Add a shameless "exported by yamdwe" note to the front page of the wiki - really shameless, but I'll keep it
```

Comment:
Oops made that while reading the code and commited it by accident :) (it was an ironic comment from me to myself - sorry about that)
Sure I will update it all.